stdlib/public/core/Builtin.swift

```diff
@@ -691,6 +691,13 @@ func _isPOD<T>(_ type: T.Type) -> Bool {
   return Bool(Builtin.ispod(type))
 }
 
+/// Returns `true` if `type` is known to refer to a concrete type during IRGen.
```

Comment:
Can you expand this doc comment with a positive and negative example, please? I know the existing comments are pretty terse, but we don't need to keep doing that.